lib/slack_bot_server/bot.rb

```diff
@@ -100,6 +100,34 @@ def say(options)
     end
   end
 
+  # Update a message on Slack
+  # @param options [Hash] a hash containing any of the following:
+  #    channel:: the name ('#general'), or the ID of the channel to send to
+  #    text:: the actual text of the message
+  #    username:: the name the message should appear from; defaults to the
+  #               value given to `username` in the Bot class definition
+  #    icon_url:: the image url to use as the avatar for this message;
+  #               defaults to the value given to `icon_url` in the Bot
+  #               class definition
```

Comment:
I believe that `message_ts` is a required option here, and `username` and `icon_url` are not valid options: https://api.slack.com/methods/chat.update

Reply:
This is fixed now, please check again.